### PR TITLE
try apply fast eventmanager

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
@gianarb could you help verify why test failure for this?

**Comment by samsonasik** [comment]
ping @gianarb

File: src/Event/FastEvmProxy.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?php
+namespace Penny\Event;
+
+use FastEventManager\EventManager;
+
+class FastEvmProxy implements EventManagerInterface
```
Comment (sensorario) [inline src/Event/FastEvmProxy.php:6]: This is just a proxy in Penny domain, but it is called FastEvmProxy. IMHO Proxy or EventProxy should be enough.

Comment (gianarb) [inline src/Event/FastEvmProxy.php:6]: Hello @sensorario 

It is a FastEventManager's proxy :)

Comment (samsonasik) [inline src/Event/FastEvmProxy.php:6]: no, there are other proxies that works for zf evm, cake evm too

Warm regards,

Abdul Malik Ikhsan

Pada 2 Feb 2016, pukul 18.55, Simone notifications@github.com menulis:

**Pull request closed without merging**